packages/twenty-chrome-extension/src/background/index.ts

```diff
@@ -1,40 +1,55 @@
 import Crypto from 'crypto-js';
 
-import { openOptionsPage } from '~/background/utils/openOptionsPage';
 import { exchangeAuthorizationCode } from '~/db/auth.db';
 import { isDefined } from '~/utils/isDefined';
 
 // Open options page programmatically in a new tab.
-chrome.runtime.onInstalled.addListener((details) => {
-  if (details.reason === 'install') {
-    openOptionsPage();
-  }
-});
+// chrome.runtime.onInstalled.addListener((details) => {
```

Comment:
Commenting this for now, will replace it with a call to a dedicated page for chrome extension of twenty documentation